examples/counters/src/index.js

```diff
@@ -24,7 +25,7 @@ if (process.env.NODE_ENV !== 'production') {
       './providers/index',
       '.'
     ], () => {
-      reloadProviders(context);
+      reloadProviders(require('./providers').default);
```

Comment:
Hot reloading probably wasn't working for you before, since you were using `react-redux-provide@5`'s `reloadProviders` arguments.  It changed to accept just the `providers` object in v6.  There's also a small chance this will need to be `reloadProviders(require('./providers'));` instead, if you end up using Babel 6's loose mode.